package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ensembl/ensembl-genome-browser",
-  "version": "0.6.3",
+  "version": "0.6.4",
```

Comment:
We are on 0.6.4 now. Sorry that my PR was late to be merged. We should use the next version number.